### ISSUE TITLE
feat(eslint-config): no unecessary destructed default

### ISSUE THREAD

**Issue by zanminkian** [zanminkian/git-validator#407]
```ts
const foo = {name: "hello"};
const {name = "hi"} = foo; // should report here but actually not
```
